.github/workflows/docker.yml

```
@@ -51,13 +49,13 @@ jobs:
         run: |
           VERSION=${GITHUB_REF#refs/*/}
           if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
-            IMAGE_TAG=$GITHUB_SHA
+            IMAGE_TAG=$GITHUB_HEAD_REF
```

I'm not sure this change is a good idea. The SHA will help to know exactly what version of the code to debug. `HEAD_REF` will not since you don't know which version of the branch was used. Is there a reason you made this change?

---

Would it help if we put this reference as a label ? ` --label "org.label-schema.vcs-ref=$GITHUB_SHA"`?

To me using the commit ref as the image tag is not very convenient as the branch name is usually more expressive of what the image/branch is meaning to solve.

---

At first, I thought you could just do something like this so you get both.

```suggestion
             IMAGE_TAG=${GITHUB_HEAD_REF}_${GITHUB_SHA}
```

I didn't know about labels. Seems good too, I'm just worry people won't know about it and therefore, won't find it. So if you choose this solution, can you document it somewhere in the developer's documentation?

---

I've added some documentation here, never too inspired about this but I think it was a shame we never mentioned there was some docker images somewhere :stuck_out_tongue:  https://github.com/hove-io/mimirsbrunn/pull/762/commits/56861baca86df1d38195bcbc72a6b1322e0d332c